tfx/orchestration/portable/base_executor_operator.py

```diff
@@ -84,6 +84,6 @@ def with_execution_watcher(
     self._execution_watcher_address = execution_watcher_address
     return self
 
-  def handle_stop(self) -> None:
+  def handle_stop(self) -> None:# noqa: B027
```

Comment:
And also a nit comment; as a coding convention, we were using two spaces before any comment `#`. This was warned when we are on the internal Google land, but I am not sure if it is captured by the linter here.

@smokestacklightnin could you let me know if this comment style is acceptable by the linter? `pylint --rcfile ./pylintrc tfx/types/system_executions.py` didn't warn me

Reply:
I don't believe the number of spaces matters. I tried with zero, one, and two spaces, and all three options passed `ruff check`.